src/kernel/caches.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2024-present The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_KERNEL_CACHES_H
+#define BITCOIN_KERNEL_CACHES_H
+
+#include <algorithm>
+#include <cstdint>
+
+//! min. -dbcache (MiB)
+static constexpr int64_t MIN_DB_CACHE{4};
+//! -dbcache default (MiB)
+static constexpr int64_t DEFAULT_DB_CACHE{450};
```

Comment:
> Mmh, is there a scenario where `DEFAULT_DB_CACHE` and `DEFAULT_KERNEL_CACHE` would ever diverge?

`node` could decide to increase the default total cache size (e.g. because it enables certain indexes by default), and that shouldn't affect the `kernel` default cache size.

> The current naming does not seem too confusing to me.

It's not necessarily confusing, but `-dbcache` is a node setting, not a kernel setting. So I think `DEFAULT_KERNEL_CACHE` is both more accurate (in `kernel/caches.h`) and - as per the previous paragraph - more futureproof.

Either way, it's easy to fix in the future. My suggestion feels more logical to me but either is fine, so can be resolved if you disagree.